src/brogue/Globals.c

```diff
@@ -2321,35 +1525,19 @@ const char itemGemsRef[NUMBER_ITEM_GEMS][30] = {
     "bloodstone",
     "jasper"
 };
-/*
-typedef struct itemTable {
-    char *name;
-    char *flavor;
-    char callTitle[30];
-    short frequency;
-    short marketValue;
-    short strengthRequired;
-    randomRange range;
-    boolean identified;
-    boolean called;
-    int magicPolarity;
-    boolean magicPolarityRevealed;
-    char description[1500];
-} itemTable;
-*/
-
-const itemTable keyTable[NUMBER_KEY_TYPES] = {
+
+itemTable keyTable[NUMBER_KEY_TYPES] = {
     {"door key",            "", "", 1, 0,   0, 0, {0,0,0}, true, false, 0, false, "The notches on this ancient iron key are well worn; its leather lanyard is battered by age. What door might it open?"},
     {"cage key",            "", "", 1, 0,   0, 0, {0,0,0}, true, false, 0, false, "The rust accreted on this iron key has been stained with flecks of blood; it must have been used recently. What cage might it open?"},
     {"crystal orb",         "", "", 1, 0,   0, 0, {0,0,0}, true, false, 0, false, "A faceted orb, seemingly cut from a single crystal, sparkling and perpetually warm to the touch. What manner of device might such an object activate?"},
 };
 
-const itemTable foodTable[NUMBER_FOOD_KINDS] = {
+itemTable foodTable[NUMBER_FOOD_KINDS] = {
     {"ration of food",      "", "", 3, 25,  0, 1800, {0,0,0}, true, false, 0, false, "A ration of food. Was it left by former adventurers? Is it a curious byproduct of the subterranean ecosystem?"},
     {"mango",               "", "", 1, 15,  0, 1550, {0,0,0}, true, false, 0, false, "An odd fruit to be found so deep beneath the surface of the earth, but only slightly less filling than a ration of food."}
 };
 
-const itemTable weaponTable[NUMBER_WEAPON_KINDS] = {
+itemTable weaponTable[NUMBER_WEAPON_KINDS] = {
```

Comment:
const has been removed for consistency with other tables, since these conceptually will be migrated to overridable-by-variant